mapsforgesrv/src/main/java/com/telemaxx/mapsforgesrv/MapsforgeSrv.java

```diff
@@ -22,102 +22,344 @@
  * 0.16.2: Check for valid tile numbers (JFBeck)
  * 0.16.3: hillshading (JFBeck)
  * 0.16.4: gamma correction (JFBeck)
- * 0.16.5: a lot of improvements/rework (non0303)
  *******************************************************************************/

```

Comment:
0.16.6: bugfix when parameter is not given and default value should be used

Reply:
@JFritzle : can you edit 2 lines(version 0.16.6) and recreate the jar file?
[0.16.6 ](https://github.com/telemaxx/mapsforgesrv/commit/95e5c14814a897745e5ea0f4cd4559cafbbf466f)